modules/service/src/main/resources/graphql/itc.graphql

```diff
@@ -4637,6 +4642,11 @@ type Query {
     input: SpectroscopyModeInput!
   ): [SpectroscopyResult!]!
 
+  spectroscopyExposureTime(
```

Comment:
I'm considering whether to have a single `exposureTime` method that will take either a spectroscopy or an imaging mode input, WDYT?

Reply:
If it simplifies the code for you, it seems fine. But, if it doesn't, I'm not sure I see a reason to combine them. The client should know what they have.